src/views/Topic.vue

```diff
@@ -1,13 +1,7 @@
 <template>
   <div class="main-container">
     <div class="main-container__cards">
-      <template v-for="topicItem in topics[`${topic}`].topics">
-        <graph-card
-          v-for="(card, cardIndex) in topicItem.graphs"
-          :key="`${topicItem.name}-${cardIndex}`"
-          :settings="card"
-        />
-      </template>
+      <graph-card v-for="(card, cardIndex) in topics[topic].cards" :key="`${cardIndex}`" :settings="card" />
```

Comment:
the key must be unique. By only using an index, then maybe something else on the page also uses the same index, so its better to give it some kind of name together with the index, as it was before